tests.py

```diff
@@ -124,8 +123,9 @@ def test_mypy_rtg_and_tests(self):
         self._test_mypy(["roam_to_git", "tests.py"])
 
     def test_mypy_all(self):
-        self._test_mypy([str(f) for f in Path(__file__).parent.iterdir()
-                         if f.is_file() and f.name.endswith(".py")])
+        self._test_mypy([str(f) for f in os.listdir(".")
+                         if os.path.isfile(f) and f.endswith(".py")
+                         and not f.startswith("setup")])
```

Comment:
Curious why you had to change this/move away from pathlib?

Reply:
It was not compatible with old versions of Python on SOME versions of Ubuntu and macOS. Life is crazy.